src/hooks/useWrapCallback.ts

```diff
@@ -50,7 +50,7 @@ export default function useWrapCallback(
                 }
               }
             : undefined,
-        inputError: sufficientBalance ? undefined : 'Insufficient ETH balance'
```

Comment:
I believe this should be "Insufficient BNB balance"

Reply:
I fix it.